templates/servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+{{ if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "docker-registry.fullname" . }}
+  labels:
+    {{- include "docker-registry.labels" . | nindent 4 }}
+spec:
+  endpoints:
+    - port: debug
+      path: /metrics
```

Comment:
Does it make sense for this path to be configurable? Or is the path here hard-coded based on what the container already provides? Are there ways to configure this chart that might make the path invalid?

Reply:
Integrated this, though checking for values in optional subdicts in helm is pita. Also setting global values from inside an if feels messy.